package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "description": "Small description for gsoc-prep-tasks goes here",
-  "license": "MIT",
+  "license": "GPLv3",
```

Comment:
https://docs.npmjs.com/files/package.json#license says to use an identifier on https://spdx.org/licenses/

GPLv3 isnt on that list.